modules/quirks/qualcomm/touchscreen-powerup.nix

```diff
@@ -0,0 +1,34 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.mobile.quirks.qualcomm;
+  inherit (lib) mkIf mkOption types;
+in
+{
+  options.mobile = {
+    quirks.qualcomm.touchscreen-powerup.enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Needed on Moto Potter to make the Synaptics touchscreen work
+      '';
+    };
+  };
+
+  config = mkIf (cfg.touchscreen-powerup.enable) {
+    systemd.services.touchscreen-powerup = {
+      description = "Setup the Synaptics touchscreen";
+      wantedBy = [ "multi-user.target" ];
+      script = ''
+        # would be better if we could detect this path somehow
+        ts_device=/sys/devices/soc/78b7000.i2c/i2c-3/3-0020
```

Comment:
Fair points, and I'll check but I'm pretty sure it will need to be a Task. For reference, https://wiki.postmarketos.org/wiki/Troubleshooting#Touch_screen_device_present_but_not_outputting_any_touch_events  lists two other Motorola devices that are said to need this fix

Reply:
I *think* the `fb-notify` hack actually does this through another "cleaner" route, but I'm not 100% sure.

Reply:
fixed in kernel, no longer needed